.circleci/config.yml

```diff
@@ -13,5483 +13,71 @@
-
-          # stable release docs push. Due to some circleci limitations, we keep
-          # an eternal PR open (#16502) for merging v1.0.1 -> master for this job.
-          # XXX: The following code is only run on the v1.0.1 branch, which might
-          # not be exactly the same as what you see here.
-          elif [[ "${CIRCLE_BRANCH}" == "v1.0.1" ]]; then
-            export COMMAND='((echo "export BUILD_ENVIRONMENT=${BUILD_ENVIRONMENT}" && echo "export GITHUB_PYTORCHBOT_TOKEN=${GITHUB_PYTORCHBOT_TOKEN}" && echo "source ./workspace/env" && echo "sudo chown -R jenkins workspace && cd workspace && . ./.circleci/scripts/cpp_doc_push_script.sh docs/stable 1.0.1") | docker exec -u jenkins -i "$id" bash) 2>&1'
-
-          # For open PRs: Do a dry_run of the docs build, don't push build
-          else
-            export COMMAND='((echo "export BUILD_ENVIRONMENT=${BUILD_ENVIRONMENT}" && echo "export GITHUB_PYTORCHBOT_TOKEN=${GITHUB_PYTORCHBOT_TOKEN}" && echo "source ./workspace/env" && echo "sudo chown -R jenkins workspace && cd workspace && . ./.circleci/scripts/cpp_doc_push_script.sh docs/master master dry_run") | docker exec -u jenkins -i "$id" bash) 2>&1'
-          fi
-
-          echo ${COMMAND} > ./command.sh && unbuffer bash ./command.sh | ts
-
-          # Save the docs build so we can debug any problems
-          export DEBUG_COMMIT_DOCKER_IMAGE=${COMMIT_DOCKER_IMAGE}-debug
-          docker commit "$id" ${DEBUG_COMMIT_DOCKER_IMAGE}
-          time docker push ${DEBUG_COMMIT_DOCKER_IMAGE}
-
-  pytorch_macos_10_13_py3_build:
+  pytorch_windows_build:
+    executor: windows-gpu-prototype
     environment:
-      BUILD_ENVIRONMENT: pytorch-macos-10.13-py3-build
-    macos:
-      xcode: "9.0"
+      BUILD_ENVIRONMENT: "pytorch-win-ws2016-cuda9-cudnn7-py3"
+      SCCACHE_BUCKET: "ossci-compiler-cache"
+      CUDA_VERSION: "10"
+      USE_CUDA: "1"
     steps:
-      # See Note [Workspace for CircleCI scripts] in job-specs-setup.yml
-      - should_run_job
       - checkout
-      - run_brew_for_macos_build
       - run:
           name: Build
-          no_output_timeout: "1h"
+          no_output_timeout: "90m"
           command: |
-            set -e
-            export IN_CIRCLECI=1
-
-            # Install sccache
-            sudo curl https://s3.amazonaws.com/ossci-macos/sccache --output /usr/local/bin/sccache
-            sudo chmod +x /usr/local/bin/sccache
-            export SCCACHE_BUCKET=ossci-compiler-cache-circleci-v2
-
-            # This IAM user allows write access to S3 bucket for sccache
+            set -ex
             set +x
-            export AWS_ACCESS_KEY_ID=${CIRCLECI_AWS_ACCESS_KEY_FOR_SCCACHE_S3_BUCKET_V4}
-            export AWS_SECRET_ACCESS_KEY=${CIRCLECI_AWS_SECRET_KEY_FOR_SCCACHE_S3_BUCKET_V4}
+            export AWS_ACCESS_KEY_ID=${MINGBO_AWS_KEY_ID}
+            export AWS_SECRET_ACCESS_KEY=${MINGBO_AWS_SECRET_KEY}
```

Comment:
To be fixed before merging right? :>

Reply:
yes, current key pair has no rights to upload image to s3